hippo4j-spring-boot/hippo4j-config-spring-boot-1x-starter/src/main/java/cn/hippo4j/config/springboot1x/starter/refresher/SpringBoot1xBootstrapConfigPropertiesBinderAdapt.java

```diff
@@ -73,7 +73,7 @@ public BootstrapConfigProperties bootstrapCorePropertiesBinder(Map<Object, Objec
         return bindableCoreProperties;
     }
 
-    private static Set<String> getNames(Object target, Iterable<String> prefixes) {
+    public static Set<String> getNames(Object target, Iterable<String> prefixes) {
```

Comment:
Why change to public?

Reply:
Because of reusing this method in the agent module.